src/App.js

```diff
@@ -4,7 +4,9 @@
  * @author Practical IT
  */
 import React, { Fragment, useState } from "react";
-import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import { Link } from "react-router-dom";
```

Comment:
any reason that Link is taken out from the existing "react-router-dom" and added here? I think it is much better to keep it in there as it will group it nicely.

Reply:
I didn't notice,,,